### PR TITLE
Changes for 2600 

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Changes for 2600 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by nabhajit-ray — APPROVED** [review]

File: examples/shared_samples/hypervisor_manager.rb [hunk 1]
```diff
@@ -51,7 +51,12 @@
 
 # Update hypervisor manager registration
 attribute = { name: @hypervisor_manager_ip }
-hm3.update(attribute)
+query_param = {}
+if @client.api_version >= 2600
+  force = true
+  query_param = { force: force }
+end
+hm3.update(attribute, query_param)
```
Comment (VenkateshRavula) [inline examples/shared_samples/hypervisor_manager.rb:59]: can update method handle query_param by default..?
https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/lib/oneview-sdk/resource.rb#L211

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+- Logical Interconnect Group
+- Network Set
+- OS Deployment Plan
+- OS Volume
+- Plan Script
+- Scope
+- Server Certificate
+- Server Hardware
+- Server Hardware Type
+- Storage Pool
```
Comment (VenkateshRavula) [inline CHANGELOG.md:34]: storage pool should be below server profile template

File: endpoints-support.md [hunk 3]
```diff
@@ -94,21 +94,21 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
+|<sub>/rest/fc-networks/bulk-delete</sub>                                                 | POST        | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | GET         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | PATCH       | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:101]: it should be heavy_minus_sign

File: endpoints-support.md [hunk 4]
```diff
@@ -117,18 +117,18 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/firmware-drivers/{id}</sub>                                                  | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:117]: why don't you validate firmware drivers for API2600.?

File: endpoints-support.md [hunk 5]
```diff
@@ -145,20 +145,20 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/interconnect-types</sub>                                                      | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:145]: validate interconnect types for api2600

File: endpoints-support.md [hunk 6]
```diff
@@ -246,14 +246,14 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
+|<sub>/rest/network-sets/{id}</sub>                                                       | PATCH    | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   |:heavy_minus_sign:  | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |
 |     **OS Deployment Plans**                                                                                                                      |
 |<sub>/rest/os-deployment-plans/</sub>                                                    | GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:258]: mark for API2600..

File: endpoints-support.md [hunk 7]
```diff
@@ -475,79 +475,79 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
 
 
 
 ## HPE Synergy Image Streamer
 
-| Endpoints                                                                         | Verb    | V300               | V500               | V600               | V800               | V1000              | V1020               |  V1600               |  V2000               |
+| Endpoints                                                                         | Verb    | V300               | V500               | V600               | V800               | V1000              | V1020               |  V1600               |  V2000               |    V2010                |
 | --------------------------------------------------------------------------------- | ------- | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: |
```
Comment (VenkateshRavula) [inline endpoints-support.md:498]: image streamer section is messed up. add a new column here

File: endpoints-support.md [hunk 8]
```diff
@@ -475,79 +475,79 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
+|<sub> /rest/build-plans/{id}</sub>                                        | GET              | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |:white_check_mark: |
+|<sub> /rest/build-plans/{id}</sub>                                        | PUT              | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |:white_check_mark: |
+|<sub> /rest/build-plans/{id}</sub>                                        | DELETE           | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |:white_check_mark: |
 |     **OS Volumes**                                                                                               |
 |<sub> /rest/os-volumes</sub>                                              | GET              | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (VenkateshRavula) [inline endpoints-support.md:542]: mark the tag for os-volume

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**